data/pg/works/Overheads.json

```diff
@@ -3,6 +3,17 @@
     "moduleName": "works",
     "Overheads": [
         {
+            "id": "1",
+            "code": "SC",
+            "description": "Supervision Charge",
+            "active": true,
+            "calculationType": "fixed rate",
+            "rate": "7.5%",
+            "effectiveFrom": 177044852,
+            "effectiveTo": null
+        },
+        {
+            "id": "2",
```

Comment:
@jagankumar-egov Some of the fields like isAutoCalculated, type, value, isWorkorderValue are required so pls add the below object for supervision charge
        {
            "id":"1",
            "code": "SC",
            "description": "Supervision Charge",
            "active": true,
            "isAutoCalculated": true,
            "type": "percentage",
            "value": "7.5",
            "isWorkOrderValue": true,
            "effectiveFrom": 177044852,
            "effectiveTo": null
        }